Source/Scene/Label.js

```diff
@@ -1030,6 +1044,42 @@ define([
                     }
                 }
             }
+        },
+
+        /**
+         * Determines whether or not run the reverseRtl algorithm on the text of the label
+         * @memberof Label.prototype
+         * @type {Boolean}
+         * @default false
+         *
+         * @example
+         * // Example 1.
+         * // Set a label's rtl during init
+         * var myLabelEntity = viewer.entities.add({
+         *   id: 'my label',
+         *   text: 'זה טקסט בעברית \n ועכשיו יורדים שורה',
+         *   rtl: true
+         * });
+         *
+         * @example
+         * // Example 2.
+         * var myLabelEntity = viewer.entities.add({
+         *   id: 'my label',
+         *   text: 'English text'
+         * });
+         * // Set a label's rtl after init
+         * myLabelEntity.rtl = true;
+         * myLabelEntity.text = 'טקסט חדש'
+         */
+        rtl : {
+            get : function() {
+                return this._rtl;
+            },
+            set : function(value) {
+                if (this._rtl !== value) {
+                    this._rtl = value;
```

Comment:
Thank you @ggetz for your comments. I will fix them after the weekend.
I have a specific question about this comment. Why to call `reverseRtl` here? If the text is written in rtl language, I don't see a reason to change the rtl flag while using it, the algorithm will work good even for ltr (left to right) languages.